data/daemon.conf

```diff
@@ -58,6 +58,12 @@ AllowEmulation=false
 # UIDs that should marked as trusted
 TrustedUids=
 
+# Vendor reports matching these globs will have releases marked as `trusted-report`, e.g.
+# * `DistroId:chromeos`
+# * `DistroId:fedora,VendorId:19`
+# * `DistroId:fedora;DistroId:rhel,DistroVersion:9`
+TrustedReports=
```

Comment:
Pandoras box.  So package maintainers will get to decide which reports are trusted?  That seems weird to me.

How about instead we have some level of opt in for the thing we're running on for trusted reports?

Reply:
> Indirectly, yes. For ChromeOS I see them just hardcoding that, and possibly the same for RHEL. For other corporate deployments I figured this would be set in an Ansible script somewhere.

I think the problem with reports being trusted just because they were reports run on a particular OS.  It's entirely possible to game this system so this seems incongruent to me.

They should be trusted because they were run by a trusted person, or trusted software stack that **signed** the report.

Reply:
Ohh it's both. Only signed reports end up in the metadata. I understand what you mean about "running on" now, i.e. we want something like $distro_id$ available as a placeholder.

Reply:
Yup!

Reply:
Okay, I've added that -- but one thing I think we also want is something like `VendorId:$self` i.e. I trust the signed report if it's been uploaded by the vendor that owns the firmware rather than trusting the any signed report from any QA team.

Reply:
Okay, done, but it depends on something like https://github.com/fwupd/fwupd/pull/5705 first.

Reply:
Okay, done.